rest-service/setup.py

```diff
@@ -73,4 +73,8 @@
     license='LICENSE',
     description='Cloudify manager rest service',
     zip_safe=False,
-    install_requires=install_requires)
+    install_requires=install_requires,
+    extras_require={
+        'dbus': ['dbus-python==1.2.4'],
```

Comment:
What's that for?

Reply:
dbus-python is used for `cfy status`; currently we do a couple `ln` calls to use the systemwide install of it, but it is nicer to simply install it into the venv instead
(current: https://github.com/cloudify-cosmo/cloudify-manager-install/blob/09472e7b91f98873925ba88ef9590cbca5ace540/cfy_manager/components/restservice/restservice.py#L102 )